### ISSUE TITLE
[Bug]: To many starting money in version 12.

### ISSUE THREAD

**Issue by 00Bla00** [OpenTTD/OpenTTD#9667]
### Version of OpenTTD

12.0 Windows 10

### Expected result

I have played TTD from the beginning, and to make the start of a new game challengin i set the maximum starting loan to 150.000 pounds. This way i started with 100.000 pounds as a loan with the option to raise the loan a bit if needed. This worked as usual in version 1.10.3 where i last startet a game.
![MaxLoan150000v10](https://user-images.githubusercontent.com/93680912/140189712-1a580b7f-df1f-4f1c-af9b-95f675cd4745.JPG)
e.

### Actual result

In version 12.0 the starting money seems to be a loan of 300.000 pounds no matter what i do. If i set the maximum loan to 150.000 i start with 300000 and the maximum loan is 450.000. If set the maximum loan to 50.000 i start with a loan of 300.000 and a maximum loan of 150.000 ! Lower than the starting loan. I have tried different currensies and different starting enviroments with the same results. Perhaps the actual starting loan can be added as a parameter next to the maximum loan to solve this ?

![MaxLoan50000v12](https://user-images.githubusercontent.com/93680912/140189732-193c4c5c-458c-4f5f-a8e2-741bd1114bf7.JPG)
.
![MaxLoan150000v12](https://user-images.githubusercontent.com/93680912/140189741-f836b36c-7dc9-4d55-ab52-d7446eac051d.JPG)
.

### Steps to reproduce

Set starting maximum loan to 50.000 pounds.
Start a new game.
See that you start with 300.000 and a max loan of 150.000

or

Set starting maximum loan to 150.000 pounds.
Start a new game.
See that you start with 300.000 and a max loan of 450.000

I hope that this bug has been proberly described.

**Comment by James103** [comment]
This works properly for me in 4487ad1 (can't test any further because no nightly builds are published after October 25).
* Starting a new game with a max loan of £150,000 starts you off with £100,000 loan out of £150,000.
* Starting a new game with a max loan of £50,000 starts you off with £50,000 loan out of £50,000.

Part of this may be due to exchange rates. Can you please go into Game Options and set "Currency" to "British Pound (GBP)" and retest with that currency?

**Comment by nielsmh** [comment]
Check if you have Inflation enabled in your game settings.

**Comment by 00Bla00** [comment]
Hi James103. The currency was set to GBP in the tests i did for the pictures. The language of the userface was Danish however.

Hi Nielsmh. Inflation was enabled for the tests. I have now tried starting games with inflation disabled and the starting money becomes right. I tried re-enabling inflation and stating again, and thenthe starting money was wrong again.
But it all works fine when inflation is disabled, and this will work fine for the games i start. Thank you.
